tests/yam/agama/boot_agama.pm

```diff
@@ -21,10 +21,18 @@ BEGIN {
 }
 use bootloader_s390;
 use bootloader_zkvm;
+use bootloader_pvm;
 
 sub run {
     my $self = shift;
 
+    # prepare kernel parameters
```

Comment:
each of the branches for s390x are doing this on its own, so this code cannot be here, it would produce unexpected results, like setting autoyast twice, the idea for special archs is to redirect to them but not do any work, needs to be exclusively a forwarding for simplicity.